Revert commit fcc0f358d17264f8956567a33471a6b52c2fc0d1

--- origsrc/gtkmm-3.18.0/gtk/gtkmm/treeview.h	2015-09-22 04:44:06.000000000 -0500
+++ src/gtkmm-3.18.0/gtk/gtkmm/treeview.h	2015-11-12 22:15:13.402433800 -0600
@@ -31,6 +31,7 @@
 
 
 #include <vector>
+#include <cstdlib>
 
 #include <gtkmm/container.h>
 #include <gtkmm/treeviewcolumn.h>
@@ -2453,7 +2454,8 @@ void _auto_store_on_cellrenderer_text_ed
       //new_value << astream; //Get it out of the stream as the numerical type.
 
       //Convert the text to a number, using the same logic used by GtkCellRendererText when it stores numbers.
-      auto new_value = static_cast<ColumnType>( std::stod(new_text) );
+      char *pchEnd = nullptr;
+      ColumnType new_value = static_cast<ColumnType>( strtod(new_text.c_str(), &pchEnd) );
 
       //Store the user's new text in the model:
       Gtk::TreeRow row = *iter;
